modules/etcd/etcd.go

```diff
@@ -0,0 +1,241 @@
+package etcd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	tcnetwork "github.com/testcontainers/testcontainers-go/network"
+)
+
+const (
+	ClientPort          = "2379"
+	PeerPort            = "2380"
+	DataDir             = "/data.etcd"
+	DefaultClusterToken = "mys3cr3ttok3n"
```

Comment:
question: do these need to be exported? I always try to keep the API surface area as small as possible.